projects/angular/src/data/tree-view/_variables.tree-view.scss

```
@@ -27,7 +28,9 @@ $clr-tree-link-hover-color: nav-variables.$clr-custom-links-hover-color !default
 $clr-tree-link-selection-color: variables.$clr-global-selection-color !default;
 $clr-tree-link-text-color: variables.$clr-color-neutral-700 !default;
 $clr-tree-checkbox-padding-top: ($clr-tree-node-touch-target - variables.$clr_baselineRem_1) * 0.5 !default;
-$clr-tree-checkbox-padding-left: ($clr-tree-node-touch-target - alert-variables.$clr-icon-dimension-sm) * 0.5 !default;
+$clr-tree-checkbox-padding-left: calc(
+  calc(#{$clr-tree-node-touch-target} - #{alert-variables.$clr-icon-dimension-sm}) * 0.5
+) !default;
```

Try to use single `calc` with parentheses. It should be identical.

```suggestion
$clr-tree-checkbox-padding-left: calc((#{$clr-tree-node-touch-target} - #{alert-variables.$clr-icon-dimension-sm}) * 0.5
) !default;
```

---

corrected in commit 3332a98